src/chapters/enums/enums-in-models.rst

```diff
@@ -0,0 +1,191 @@
+Enums in Model Classes
+======================
+
+One application of enum types is to represent categories of objects. We will take this approach in our ``coding-events`` application to categorize events based on their type, such as *conference* or *meetup*.
+
+Enum Types in Models - Video
+----------------------------
+
+.. TODO: add video
+
+.. admonition:: Note
+
+   If you ever want to verify what code the video starts with, check out the `display-error-messages <https://github.com/LaunchCodeEducation/CodingEventsDemo/tree/display-error-messages>`__ branch in ``CodingEventsDemo``.
+   If you ever want to verify what code the video ends with, check out the `enums <https://github.com/LaunchCodeEducation/CodingEventsDemo/tree/enums>`__ branch in ``CodingEventsDemo``.
+
+Enum Types in Models - Text
+---------------------------
+
+Create an Enum Class
+^^^^^^^^^^^^^^^^^^^^
+
+In your ``Models`` directory within ``CodingEvents``, create a new class called ``EventType``. 
+Replace the ``class`` keyword with ``enum`` and delete the default constructor.
+
+Because enum values are constants, we use naming conventions and write them in all caps.
+Each value is demarcated with a comma and the list is completed with a semicolon.
```

Comment:
A semicolon is not needed in C#. This sentence is probably a carry-over from Java.

Reply:
Done!